{"name": "Add New Employee", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps\\user_login.py\", line 7, in login\n    context.playwright = sync_playwright().start()\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "steps": [{"name": "Given I navigate to the login page", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"steps\\user_login.py\", line 7, in login\n    context.playwright = sync_playwright().start()\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\a90011969\\Playwright-Python\\Playwright-Python\\playwright-behave-env\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "start": 1739258905575, "stop": 1739258905578}, {"name": "And I enter valid credentials", "status": "skipped", "start": 1739258905579, "stop": 1739258905579}, {"name": "When I click on PIM and then Add option", "status": "skipped", "start": 1739258905579, "stop": 1739258905579}, {"name": "And I enter valid details of the new employee", "status": "skipped", "start": 1739258905579, "stop": 1739258905579}, {"name": "Then a new employee should be added", "status": "skipped", "start": 1739258905579, "stop": 1739258905579}], "start": 1739258905574, "stop": 1739258905579, "uuid": "d1a31fde-9584-4ab3-ac72-100b72c67537", "historyId": "96dfa068031ec2ea1de9940528156079", "fullName": "User Login: Add New Employee", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}